spec/features/browsing_item_spec.rb

```diff
@@ -40,7 +40,7 @@
     end
 
     it 'updates the link to the ARK with a fragment identifier for the UV' do
-      expect(page).to have_selector '.electronic-access a[href$="3395923#view"]', text: 'Digital content'
+      expect(page).to have_selector '.availability--online a[href$="7044622#view"]', text: 'Table of contents'
```

Comment:
Question: Why did this change?

Reply:
I had the same question. I couldn't get to the href from the .electronic-access class. Only from .availability--online class. 

<div class="availability--online">
    <h3>Available Online</h3>
    <ul><ul><li class="electronic-access"></li><li><a href="/catalog/7044622#view">Table of contents</a></li><li class="electronic-access"></li><li><a href="/catalog/7044622#view">Digital content</a></li><li id="full_text" class="availability--panel availability_full-text availability_full-text-alternative" data-umlaut-services="true"></li></ul><li class="electronic-access"></li><li><a target="_blank" href="https://na05.alma.exlibrisgroup.com/view/uresolver/01PRI_INST/openurl?u.ignore_date_coverage=true&amp;portfolio_pid=53837284010006421&amp;Force_direct=true"></a> </li><li class="electronic-access"></li><li><a target="_blank" href="https://na05.alma.exlibrisgroup.com/view/uresolver/01PRI_INST/openurl?u.ignore_date_coverage=true&amp;portfolio_pid=53837284030006421&amp;Force_direct=true"></a> </li></ul>
  </div>